nullaway/src/test/java/com/uber/nullaway/CoreTests.java

```diff
@@ -1154,4 +1155,38 @@ public void enclosingExpressionForNewClassTree() {
             "}")
         .doTest();
   }
+
+  //  @Ignore
+  @Test
+  public void unboxForEachLoop() {
```

Comment:
This test belongs in `JSpecifyArrayTests`.  It should only report an error in JSpecify mode.